atomic_reactor/plugins/post_push_to_pulp.py

```diff
@@ -345,7 +345,12 @@ def run(self):
                 image_file.flush()
                 crane_repos = self.push_tar(image_file.name, image_names)
 
-        for image_name in crane_repos:
-            self.log.info("image available at %s", str(image_name))
+        if not hasattr(self.workflow, "image_names_printed") or \
+           not self.workflow.image_names_printed:
```

Comment:
Implemented this.

I had to hardcode 'pulp_push' plugin key in order to prevent cyclic imports, not sure if there is a better way to do this